Core/Types/CkanModule.cs

```diff
@@ -623,6 +623,22 @@ public string DescribeInstallStanzas()
             return string.Join(", ", descriptions);
         }
 
+        /// <summary>
+        /// Return an archive.org URL for this download, or null if it's not there.
+        /// The filenames look a lot like the filenames in Net.Cache, but don't be fooled!
+        /// Here it's the first 8 characters of the SHA1 of the DOWNLOADED FILE, not the URL!
+        /// </summary>
```

Comment:
Yeah, the hash of that file will remain consistent. The url may not.
```
Produces a filename based of the first 8 digits in sha1 hash,
the 'identifier' and the 'version' in the metadata if the
download_hash exists. Returns '0' if there is no download hash
or has an content type other than zip/gz/tar/tar.gz.
```
There are some [tests](https://github.com/KSP-CKAN/NetKAN-bot/blob/master/t/App/KSP_CKAN/Metadata/Ckan.t#L71) that ensure the correct filenames are generated.

Reply:
Yup, either way makes enough sense to me. I just wanted to note it explicitly since we have two different 8-digit hexadecimal filename prefixes floating around, and it's not easy to tell that they're (supposed to be) different at a glance.

Reply:
Yeah, actually makes sense for URL in the cache. If the url changes, you probably do want to re-download.